apps/validate/validate.py

```diff
@@ -79,6 +79,17 @@
     'usageterms': lazy_gettext('Usage Terms'),
 }
 
+ERROR_MESSAGES = {
+    'REQUIRED_FIELD_MESSAGE': lazy_gettext('is a required field'),
```

Comment:
I would use the constants so just `REQUIRED_FIELD` as key and also later when using it `ERROR_MESSAGES[REQUIRED_FIELD]`